src/importsource.cpp

```diff
@@ -287,6 +287,17 @@ bool ImportSource::removeAllUnits()
     return status;
 }
 
+bool ImportSource::doEquals(const EntityPtr &other) const
+{
+    if (Entity::doEquals(other)) {
+        auto importSource = std::dynamic_pointer_cast<ImportSource>(other);
+        if (importSource != nullptr) {
+            return mPimpl->mUrl == importSource->url();
+        }
+    }
+    return false;
+}
```

Comment:
When testing for equal import sources, does it matter what they contain or only what they define?  ie
impA -> fileA
impB -> fileA
componentC, componentD both imported from impA
componentE imported from impB
Are impA and impB still equal?

Reply:
Imports only describe where the information is coming from. So yes impA and impB are still equal.